nibabel/xmlutils.py

```diff
@@ -27,7 +27,11 @@ def _to_xml_element(self):
     def to_xml(self, enc='utf-8'):
         """ Output should be an xml string with the given encoding.
         (default: utf-8)"""
-        return tostring(self._to_xml_element(), enc)
+        ele = self._to_xml_element()
+        if ele is None:
```

Comment:
My eye finds the new line less clumsy and easier to read, but no, it's OK, for stylistic stuff like this, you're welcome to push back.